sky/data/storage.py

```diff
@@ -1250,7 +1250,8 @@ def _delete_gcs_bucket(self, bucket_name: str) -> None:
             with backend_utils.safe_console_status(
                     f'[bold cyan]Deleting [green]bucket {bucket_name}'):
                 if num_files >= _GCS_RM_MAX_OBJS:
-                    remove_obj_command = f'gsutil -m rm -a gs://{bucket_name}/*'
+                    remove_obj_command = ('gsutil -m rm -a'
```

Comment:
Is there a particular reason why we don't use `gsutil -m rm -r gs://bucket`, as suggested by [gsutil docs](https://cloud.google.com/storage/docs/gsutil/commands/rm)?
![image](https://user-images.githubusercontent.com/4416605/197464412-9e985b15-730b-451e-994f-1c7a429e63b8.png)

Reply:
I just tried this command... it works for all cases. Thanks Romil!